src/test/ui/stability-attribute/generics-default-stability.rs

```diff
@@ -0,0 +1,264 @@
+// ignore-tidy-linelength
+// aux-build:unstable_generic_param.rs
+#![feature(unstable_default6)]
```

Comment:
Should that `6` be there?

Reply:
Yep, the it's for enabling the feature flagged type parameter on `Struct6`, `Alias6`, `Enum6`. Maybe it could have a better name though. 

https://github.com/rust-lang/rust/pull/77118/files#diff-6497816ecc9b7bf6b434066691e37224R58
https://github.com/rust-lang/rust/pull/77118/files#diff-6497816ecc9b7bf6b434066691e37224R121
https://github.com/rust-lang/rust/pull/77118/files#diff-6497816ecc9b7bf6b434066691e37224R164